backend/package.json

```diff
@@ -7,6 +7,8 @@
 		"test": "echo \"Error: no test specified\" && exit 1",
 		"build": "tsc",
 		"db:build": "serverless invoke local --function dbInit",
+		"db:drop": "serverless invoke local --function dbDrop",
```

Comment:
You don't need a drop and alter you can just use force: true in the sync function

read this

https://sequelize.org/master/class/lib/sequelize.js~Sequelize.html#instance-method-sync

Reply:
It still fails when updates related to table references.